launch_ros/launch_ros/utilities/to_parameters_list.py

```diff
@@ -45,8 +49,19 @@ def to_parameters_list(
     for params_set_or_path in evaluated_parameters:
         if isinstance(params_set_or_path, pathlib.Path):
             with open(str(params_set_or_path), 'r') as f:
+                if namespace is not None:
+                    node_name = f"{namespace}/{node_name}"
+                node_name_with_slash = f"/{node_name}"
+                param_dict = yaml.safe_load(f)
+                try:
+                    param_dict = param_dict[node_name]["ros__parameters"]
+                except KeyError:
+                    try:
+                        param_dict = param_dict[node_name_with_slash]["ros__parameters"]
+                    except KeyError:
+                        pass
```

Comment:
@rebecca-butler hmm, a few questions about behavior:

- What if no node name was specified for the node on launch?
- What if there's no entry in the parameter file for that node name? 
- What if a wildcard was used in the file?